lib/index.js

```diff
@@ -261,7 +261,7 @@ class Service extends AdapterService {
 
     // Force the {raw: false} option as the instance is needed to properly
     // update
-    const updateOptions = Object.assign({}, params.sequelize, { raw: false });
+    const updateOptions = Object.assign({}, params.sequelize, params, { raw: false });
```

Comment:
I'm not sure this works. `params` is an object shared between hooks and is often a dumping ground for all sorts of data related to a service call. For example, I might have a hook at the beginning of a service call which sets some state to be used by other hooks later in the service pipeline. The `params` object is where this information is stored. Any information/data used in db queries should have a meaningful place - usually `params.query` and `params.sequelize`. If we need to designate other properties for holding query-specific data, we can... but using the entire `params` objects is not the solution IMO.